config.xml

```diff
@@ -11,36 +11,52 @@
         <package name="7zip-15-05.vm"/>
         <package name="apimonitor.vm"/>
         <package name="apktool.vm"/>
+        <package name="bytecodeviewer.vm"/>
         <package name="capa.vm"/>
         <package name="cmder.vm"/>
+        <package name="codetrack.vm"/>
+        <package name="common.vm"/>
```

Comment:
so it is needed here, I think there was a question on this before

Reply:
almost, this was the question: https://github.com/mandiant/flare-vm/issues/453#issuecomment-1565498611

Reply:
it is not, it is a dependency of all packages. I'll remove it! 👍